mypy/test/data.py

```diff
@@ -556,6 +556,9 @@ def repr_failure(self, excinfo: Any) -> str:
 
 
 class DataSuite:
+    def __init__(self, *, update_data: bool = False) -> None:
```

Comment:
I've added it here since I believe that at the end of the road no class will need it, or any other argument;. So for the time being, a prototype `__init__` in the base class (just for that keyword) should suffice.